.github/workflows/livetest.yml

```diff
@@ -0,0 +1,62 @@
+name: Live test
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  live-test:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/livetest')
+    name: Live test PRs
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y cockpit-system cockpit-ws nodejs npm sassc make
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          # need this to also fetch tags
+          fetch-depth: 0
+
+      - name: Compile and install cockpit
+        run: make && sudo make install
+
+      - name: Create a new test user
+        run: sudo useradd livetest; echo livetest:foobar | sudo chpasswd
+
+      - name: Get AWS port to forward to
+        id: remote_port
+        env:
+          ISSUE_NUMBER: "${{ github.event.issue.number }}"
+        run: |
+            echo ::set-output name=AWS_PORT_NUMBER::$(($ISSUE_NUMBER % 100 + 15000))
+
+      - name: Write a comment on the PR with link to the cockpit instance
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'Cockpit preview: https://logs.cockpit-project.org:${{ steps.remote_port.outputs.AWS_PORT_NUMBER }}'
+            })
+
+      - name: Create an SSH private key from the secret
+        run: |
+          echo "${{ secrets.SINK_SSHRELAY_PRIVATE }}" > /tmp/SINK_SSHRELAY_PRIVATE
+          chmod 600 /tmp/SINK_SSHRELAY_PRIVATE
+          mkdir -p ~/.ssh &&  ssh-keyscan -H logs.cockpit-project.org > ~/.ssh/known_hosts
+
+      - name: Forward cockpit port 9090 to a server we can access
+        run: ssh -tt -R ':${{ steps.remote_port.outputs.AWS_PORT_NUMBER }}':localhost:9090  -i /tmp/SINK_SSHRELAY_PRIVATE sshgw@logs.cockpit-project.org 'sleep infinity' &
+
+      - name: Delete security keys
+        run: rm /tmp/SINK_SSHRELAY_PRIVATE
+
+      - name: Block workflow from finishing
+        run: sleep infinity
```

Comment:
Hmm, I wonder if we should have a better way of ending it, as like this it will always time out and generate an error email. The simplest would probably to `sleep 4h`.